### PR TITLE
Issue58 new naming

### PR DESCRIPTION
**Pull request by andersbonielsen**
FIX: spaces and dots have been removed from the naming of folders, files and URIs.
Tiny update to README.md mostly due to link to new folder with new name without space.

We are trying to follow the recommendations and rules given in these documents:

https://www.w3.org/2013/dwbp/wiki/URI_Design_and_Management_for_Persistence
https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/document/10-rules-persistent-uris
https://www.rfc-editor.org/rfc/rfc3986
https://www.rfc-editor.org/rfc/rfc8820

### CODE REVIEW

**Review by karinbredenberg — COMMENTED** [review]
I approve the change but it needs to be checked if we need to do something in the DNS before it can be merged to master.

File: README.md [hunk 1]
```diff
@@ -25,21 +25,28 @@ The development and release of new versions will be coordinated in the DILCIS bo
...
+It has been developed by the DILCIS Board during the E-ARK3 project.
+
 ## Specifications
 
 - SIARD 1.0   [2008-09-30](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%201.0/format/2008-09-30/)
 - SIARD 2.0   [2016-06-01](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%202.0/format/2016-06-01/)
 - SIARD 2.1   [2018-02-15](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%202.1/format/2018-02-15/)
 - SIARD 2.1.1 [2019-05-15](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%202.1/format/2018-02-15/)
-- SIARD 2.2   [2021-08-31](./specification/SIARD_2_2.pdf)
+- SIARD 2.2   [2021-08-31](https://github.com/DILCISBoard/SIARD/blob/master/SIARD_2-2/format/2021-08-31/)
```
Comment (carlwilson) [inline README.md:38]: Is the space in this URL supposed to be here or should it be encoded?

**Review by carlwilson — COMMENTED** [review]
I'm pretty sure we are OK DNS-wise but I'm unsure about the form of the URL in the README. See comment inline.

**Comment by andersbonielsen** [comment]
Hello Carl,

There is no and should be no space, it is an underscore.
SIARD[underscore]2-2/format

https://github.com/DILCISBoard/SIARD/blob/master/SIARD_2-2/format/2021-08-31/)



Fra: Carl Wilson ***@***.***>
Sendt: 27. november 2023 14:37
Til: DILCISBoard/SIARD ***@***.***>
Cc: Anders Bo Nielsen ***@***.***>; Author ***@***.***>
Emne: Re: [DILCISBoard/SIARD] Issue58 new naming (PR #60)


@carlwilson commented on this pull request.

I'm pretty sure we are OK DNS-wise but I'm unsure about the form of the URL in the README. See comment inline.

________________________________

In README.md<https://github.com/DILCISBoard/SIARD/pull/60#discussion_r1406168344>:

>  ## Specifications



 - SIARD 1.0   [2008-09-30](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%201.0/format/2008-09-30/)

 - SIARD 2.0   [2016-06-01](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%202.0/format/2016-06-01/)

 - SIARD 2.1   [2018-02-15](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%202.1/format/2018-02-15/)

 - SIARD 2.1.1 [2019-05-15](https://github.com/DILCISBoard/SIARD/blob/master/SIARD%202.1/format/2018-02-15/)

-- SIARD 2.2   [2021-08-31](./specification/SIARD_2_2.pdf)

+- SIARD 2.2   [2021-08-31](https://github.com/DILCISBoard/SIARD/blob/master/SIARD_2-2/format/2021-08-31/)

Is the space in this URL supposed to be here or should it be encoded?

**Comment by andersbonielsen** [comment]
Hello Carl, @carlwilson 
Do you still see a problem with this pull request?

**Review by carlwilson — APPROVED** [review]

**Comment by carlwilson** [comment]
> Hello Carl, @carlwilson Do you still see a problem with this pull request?

Not at all, and have now approved.

**Pull request merged**